backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/Task.java

```diff
@@ -152,6 +153,10 @@ public T setTotal(int total) {
   }
 
   public T start() {
+    if (this.isDisabled()) {
```

Comment:
as discussed this morning, we don't need this I suppose if 'disabled' means 'not running via cron'